web/src/lib/components/AuctionEditor.svelte

```diff
@@ -51,12 +57,17 @@
                     </label>
                     <input bind:value={auction.title} type="text" name="title" class="input input-bordered w-full max-w-xs" />
                 </div>
+                <NavTab {activeItem} {items} on:tabChange={tabChange}/>
+                {#if activeItem === 'Description'}
                 <div class="form-control">
                     <label class="label" for="description">
                         <span class="label-text">Description</span>
                     </label>
                     <textarea bind:value={auction.description} rows="4" class="textarea textarea-bordered h-24" placeholder=""></textarea>
                 </div>
+                {:else if activeItem === 'Preview'}
```

Comment:
So the rendering is done by the svelte component, not by the server?

Reply:
Yes the component renders the markdown